pydevices/HtsDevices/acq2106_435sc.py

```diff
@@ -81,80 +82,83 @@ def getUUT(self):
         uut = acq400_hapi.Acq2106(self.node.data(), monitor=False, has_wr=True)
         return uut
 
+    def computeGains(self, g):
+        g1opts = [ 1000, 100, 10, 1 ]
+        g2opts = [ 1, 2, 5, 10 ]
 
-    def setGainsOffsets(self, card):
+        for g1opt in g1opts:
```

Comment:
We should add a TODO to this:
I want to support explicitly setting both gains in the form of
* single number -> gain auto configuration
* two numbers separated by a comma -> set gain1 and gain2 explicitly